lms/static/js/instructor_dashboard/ecommerce.js

```diff
@@ -27,6 +27,11 @@ var edx = edx || {};
     });
 
     $(function() {
+        var registration_code_status_form = $("form#set_regcode_status_form");
```

Comment:
jQuery variables are usually prefixed with `$` (e.g. `$registration_code_status_form`). Also, these can be combined into a single call:

```javascript
var $registration_code_status_form = $("form#set_regcode_status_form"),
    $lookup_button = $('#set_regcode_status_form input#lookup_regcode'),
    $registration_code_status_form_error = $('#set_regcode_status_form #regcode_status_form_error'),
    $registration_code_status_form_success = $('#set_regcode_status_form #regcode_status_form_success');
```

Going even further, you can use context (https://api.jquery.com/jquery/#jQuery1):

```javascript
var $registration_code_status_form = $("form#set_regcode_status_form"),
    $lookup_button = $('#lookup_regcode', $registration_code_status_form),
    $registration_code_status_form_error = $('#regcode_status_form_error', $registration_code_status_form),
    $registration_code_status_form_success = $('#regcode_status_form_success', $registration_code_status_form);
```